src/main/java/city/parking/LoadDatabase.java

```diff
@@ -0,0 +1,39 @@
+package city.parking;
+
+import city.parking.entities.Money;
+import city.parking.entities.ParkingProcess;
+import city.parking.repositories.ParkingProcessRepository;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.Currency;
+
+@Configuration
+@Slf4j
+public class LoadDatabase {
```

Comment:
To nie powinno być w klasach src/main/java. Jak można inaczej zainicjować bazę na potrzeby testów?

Reply:
Za pomocą pliku src/main/resources/data.sql, można tam utworzyć potrzebne 'parking procesy'